src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,285 @@
+Feature: Descriptors are a set of mechanisms to support flexible enumerations or code sets
+
+        Background:
+            Given the system has this "educationOrganizationCategoryDescriptors"
+                  | codeValue | description | namespace                                               | shortDescription |
+                  | School    | School      | uri://ed-fi.org/EducationOrganizationCategoryDescriptor | School           |
+              And the system has this "disabilityDescriptors"
+                  | codeValue       | description          | namespace                            | shortDescription     |
+                  | Disability Test | User Disability Test | uri://ed-fi.org/DisabilityDescriptor | User Disability Test |
+
+        @ignore
+        Scenario: 01 / Ensure clients can create a Descriptor with the required attributes
+             When a POST reques is made to "/ed-fi/disabilityDescriptors" with
+                  """
+                  {
+                      "namespace": "uri://ed-fi.org/DisabilityDescriptor",
+                      "codeValue": "Deaf-Blindness",
+                      "shortDescription": "Deaf-Blindness",
+                      "description": "Deaf-Blindness"
```

Comment:
Should we remove `description` since it is not a required attribute?

Reply:
It would be better to remove it to avoid any confusions with the mandatory values.